src/accelerate/commands/config/default.py

```diff
@@ -62,6 +62,14 @@ def write_basic_config(mixed_precision="no", save_location: str = default_json_c
             config["distributed_type"] = "MULTI_GPU"
         else:
             config["distributed_type"] = "NO"
+    if is_xpu_available():
+        num_gpus = torch.xpu.device_count()
```

Comment:
```suggestion
        num_xpus = torch.xpu.device_count()
```
Since they are called XPU everywhere...